source/Gui/SimulationParametersWindow.h

```diff
@@ -41,7 +41,14 @@ class _SimulationParametersWindow : public _AlienWindow
     std::optional<int> _sessionId;
     bool _focusBaseTab = false;
     std::vector<std::string> _cellFunctionStrings;
-
+    
+    std::vector<std::string> _spotNameStrings;
+    std::vector<std::string> _spotTabIDs;
+    std::vector<std::string> _copiedSpotNameStrings;
+    std::vector<std::string> _copiedSpotTabIDs;
+    
+    int _serialTabID;
```

Comment:
I would prefer to initialize PODs at the point where they are declared. So here it should probably be `= 0`.